cordapp-cpk2/src/main/java/net/corda/plugins/cpb2/CpbTask.java

```diff
@@ -55,6 +58,8 @@ public CpbTask() {
             m.getAttributes().put(CPB_NAME_ATTRIBUTE, getArchiveBaseName());
             m.getAttributes().put(CPB_VERSION_ATTRIBUTE, getArchiveVersion());
         });
+
+        cpkNames = new HashMap<>();
```

Comment:
I'm not a gradle expert, but having a static you reset on each instantiation of this type strikes me as a bit odd. Could you explain it more?

Reply:
I'm not entirely sure why but for some reason when I don't reset it then it triggers the error. I think it might be persisting through multiple calls of the same plugin within the same project

Reply:
I think what we are doing here (resetting it) suggests it should be made an instance property. 

However it's interesting. I would expect this process to be one-off? I.e. pack CPKs intended for a single CPB and then exits? What more CPKs are there being processed/ output-ed by the prints added?

Reply:
That fixed the issue, thanks